modules/snooker/src/test/ts/module/ephox/snooker/test/Assertions.ts

```diff
@@ -47,11 +60,44 @@ const checkOld = (
   Bars.destroy(wire);
 };
 
+const checkOldMultitude = (
+  expCell: { section: number; row: number; column: number },
+  expectedHtml: string,
+  input: string,
+  operation: OperationMultile<TargetSelection>,
+  paths: TargetLocation[]
+) => {
+  const table = SugarElement.fromHtml<HTMLTableElement>(input);
+  Insert.append(SugarBody.body(), table);
+  const wire = ResizeWire.only(SugarBody.body());
+  const result = operation(wire, table,
+    {
+      selection: Arr.map(paths, (path) =>
+        Hierarchy.follow(table, [ path.section, path.row, path.column, 0 ]).getOrDie()
+      )
+    },
+    Bridge.generators
+  );
+
+  const actualPath = Hierarchy.path(table, result.getOrDie().cursor.getOrDie()).getOrDie('could not find path');
+  assert.eq([ expCell.section, expCell.row, expCell.column ], actualPath);
+
+  // Presence.assertHas(expected, table, 'checking the operation on table: ' + Html.getOuter(table));
```

Comment:
Was there a reason this is commented out? If it's not useful, then should it just be deleted?

Reply:
Because it tagged along from `checkOld`, and I forgot to remove it.